source/braille.py

```diff
@@ -1190,35 +1196,58 @@ def getFormatFieldBraille(field, fieldCache, isAtStart, formatConfig):
 
 
 def _getFormattingTags(
-	field: dict[str, str], fieldCache: dict[str, str], formatConfig: dict[str, bool]
+	field: dict[str, str],
+	fieldCache: dict[str, str],
+	formatConfig: dict[str, bool],
 ) -> str | None:
+	"""Get the formatting tags for the given field and cache.
+
+	Formatting tags are calculated according to the preferences passed in :param:`formatConfig`.
+
+	:param field: The format current field.
+	:type field: dict[str, str]
+	:param fieldCache: The previous format field.
+	:type fieldCache: dict[str, str]
+	:param formatConfig: The user's formatting preferences.
+	:type formatConfig: dict[str, bool]
+	:return: The braille formatting tag as a string, or None if no pertinant formatting is applied.
+	:rtype: str | None
```

Comment:
Typo fix, types aren't needed, they should be in docstring headers

https://github.com/nvaccess/nvda/blob/master/projectDocs/dev/codingStandards.md#Docstrings

```suggestion
	:param field: The current format field.
	:param fieldCache: The previous format field.
	:param formatConfig: The user's formatting preferences.
	:return: The braille formatting tag as a string, or None if no pertinent formatting is applied.
```